install

```diff
@@ -389,15 +417,14 @@ Dir.chdir HOMEBREW_REPOSITORY do
     # pipefail to cause the exit status from curl to propagate if it fails
```

Comment:
Could consider eliminating the `curl`-only path entirely if it's requiring a modern HTTPS client anyway?

Reply:
If it was only there for use by older OSes, then yeah I guess.

Reply:
@EricFromCanada I think it's there originally for OSs where they didn't have Git preinstalled but HTTPS from GitHub still worked. Now that Git is more backwards compatible than Curl for accessing GitHub I think added complexity isn't worth it.